scalableminds/webknossos
# Remove mouse inversion option

### Mailable description of changes:
 - [TracingView] The mouse movement inversion configuration option has been removed.

### Steps to test:
- Open tracing view
- Check the mouse movement and zooming still works correctly

### Issues:
- fixes #2249 

------
